src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.Tests/ServiceControllerTests.cs

```diff
@@ -87,9 +87,16 @@ private static bool RunningWithElevatedPrivileges
             get { return s_runningWithElevatedPrivileges.Value; }
         }
 
-        private void AssertExpectedProperties(ServiceController testServiceController)
+        private void AssertExpectedProperties(ServiceController testServiceController, bool serviceNameToUpper = false)
         {
-            Assert.Equal(_testService.TestServiceName, testServiceController.ServiceName);
+            if (serviceNameToUpper)
```

Comment:
For example this one: https://github.com/dotnet/corefx/pull/20466/files#diff-98c153637fe0b790f1745fd716106741R109

Which creates the ServiceController with lowercase value.